class/Defaults/Recipient/Email.php

```diff
@@ -40,8 +40,22 @@ public function parse_value( $value = '' ) {
 			$value = $this->get_default_value();
 		}
 
+		/**
+		 * Include 'filter-id:your-favorite-id' in value to specify a filter id.
+		 *
+		 * defaults to 'email' (ie. filter 'notification/recipients/email'):
+		 */
+		$filter_id = 'email';
+
+		if ( preg_match( "/\bfilter-id:([\w][\w-]*)/", $value, $matches ) ) {
+			$filter_id = $matches[1];
+			$value = trim( preg_replace( "/\bfilter-id:[\w][\w-]*/", "", $value ) );
+		}
+
+		$value  = apply_filters( 'notification/recipients/' . $filter_id, $value );
```

Comment:
```suggestion
		$value  = apply_filters( 'notification/recipient/email/' . $filter_id, $value );
```

The reason for a change is to include the recipient slug in the filter as this applies only to the Email recipient. Maybe would be good to change the default `email` to just `default` this case?

This would require also changing the comment on line 46 and FAQ answer